src/frontend/src/selenium.test.ts

```diff
@@ -387,6 +416,30 @@ async function run_in_browser_common(
   }
 }
 
+async function fillText(driver: ThenableWebDriver, id: string, text: string) {
+  const elem = await driver.findElement(By.id(id));
+  elem.sendKeys(Key.CONTROL + "a");
+  elem.sendKeys(Key.DELETE);
+  elem.sendKeys(text);
+}
+
+async function switchToPopup(driver: ThenableWebDriver) {
+  let handles = await driver.getAllWindowHandles();
+  expect(handles.length).toBe(2);
+  await driver.switchTo().window(handles[1]);
```

Comment:
Is this order always deterministic? Could we maybe do something like finding the popup via its window title?

Reply:
So far it works 🤷🏻

Reply:
Allright, good enough for me. If it ever starts failing the test will fail local enough, that we can figure it out.